docs/partners/1-badging/available-badges/ed-fi-implementation-partner-badge.md

```diff
@@ -0,0 +1,66 @@
+# Ed-Fi Implementation Partner Badge
+
+![Ed-Fi Implementation Partner Badge](../img/Ed-Fi%20Implementation%20Partner%20Badge-Watermarked.png)
+
+To be awarded this badge, a vendor must be able to demonstrate adherence to
+Ed-Fi best practices and their services must facilitate Ed-Fi data standards
+delivery in their customer's solutions.
+
+## Requirements
+
+* **Verifications - Quality**
+    Like other badges, a verifying agency/company is required. The nature of the
+    verification is important due to the custom nature of offering
+    implementation services.
+  * The verified vendor implementation must also be badged/certified to
+        qualify as a verification. This is required to ensure that
+        implementation services are following best practices and current
+        certification/badging requirements.
+  * Collaboratives or regional service centers can provide verification if
+        their solution is badged as a “Ed-Fi Managed ODS API Platform Badge”
+  * Ed-Fi may use a developer to look at code of the implementation to look
+        for best practices (e.g. look for multi-threading in code to support
+        best practice)
+  * Verifications from individual LEAs\\SEAs require a deeper validation
+        than other means to ensure adherence to Ed-Fi standards and
+        repetability.  Vendors should consider productizing their service for
+        more streamlined scale-ability.  Part of this verification includes
+        validation Ed-Fi core components are used, code inspection to validate
+        that best practices are being used, and reviewing processes exist for
+        repeatability.
+* **Verifications - Quantity**
+    Unlike other badges, the Implementation Partner badge requires two
+    verifications. Due to the custom nature of implementations Ed-Fi requires a
+    second verification to demonstrate repeatability in the marketplace.
+* **Proof of repeatability
+    **Implementation partners must demonstrate their services are “productized”
```

Comment:
The `**` on this line needs to move to the end of the line above. Currently this is not correctly styling the text as bold. Same on a few more below this.